framework/src/test/java/org/tron/core/WalletMockTest.java

```diff
@@ -0,0 +1,1330 @@
+  }
+
+  @After
+  public void  clearMocks() {
+    Mockito.clearAllCaches();
+  }
+
+  @Test
+  public void testSetTransactionNullException() throws Exception {
+    Wallet wallet = new Wallet();
+    TransactionCapsule transactionCapsuleMock
+        = mock(TransactionCapsule.class);
+
+    Method privateMethod = Wallet.class.getDeclaredMethod(
+        "setTransaction", TransactionCapsule.class);
+    privateMethod.setAccessible(true);
+    privateMethod.invoke(wallet, transactionCapsuleMock);
+
+    assertTrue(true);
+  }
+
+  @Test
+  public void testCreateTransactionCapsuleWithoutValidateWithTimeoutNullException()
+      throws Exception {
+    Wallet wallet = new Wallet();
+    com.google.protobuf.Message message =
+        mock(com.google.protobuf.Message.class);
+    Protocol.Transaction.Contract.ContractType contractType =
+        mock(Protocol.Transaction.Contract.ContractType.class);
+    long timeout = 100L;
+
+    try (MockedConstruction<TransactionCapsule> mocked = mockConstruction(TransactionCapsule.class,
+        (mock, context) -> {
+          when(mock.getInstance()).thenReturn(Protocol.Transaction.newBuilder().build());
+        })) {
+      Method privateMethod = Wallet.class.getDeclaredMethod(
+            "createTransactionCapsuleWithoutValidateWithTimeout",
+            com.google.protobuf.Message.class,
+            Protocol.Transaction.Contract.ContractType.class,
+            long.class);
+      privateMethod.setAccessible(true);
+      privateMethod.invoke(wallet, message, contractType, timeout);
+      assertTrue(true);
+    }
+  }
+
+  @Test
+  public void testCreateTransactionCapsuleWithoutValidateWithTimeout()
+      throws Exception {
+    Wallet wallet = new Wallet();
+    com.google.protobuf.Message message =
+        mock(com.google.protobuf.Message.class);
+    Protocol.Transaction.Contract.ContractType contractType =
+        mock(Protocol.Transaction.Contract.ContractType.class);
+    long timeout = 100L;
+    BlockCapsule.BlockId blockId = new BlockCapsule.BlockId();
+
+    try (MockedConstruction<TransactionCapsule> mocked = mockConstruction(TransactionCapsule.class,
+        (mock, context) -> {
+          when(mock.getInstance()).thenReturn(Protocol.Transaction.newBuilder().build());
+        })) {
+      ChainBaseManager chainBaseManagerMock = mock(ChainBaseManager.class);
+
+      Field field = wallet.getClass().getDeclaredField("chainBaseManager");
+      field.setAccessible(true);
+      field.set(wallet, chainBaseManagerMock);
+
+      when(chainBaseManagerMock.getHeadBlockId()).thenReturn(blockId);
+
+      Method privateMethod = Wallet.class.getDeclaredMethod(
+          "createTransactionCapsuleWithoutValidateWithTimeout",
+          com.google.protobuf.Message.class,
+          Protocol.Transaction.Contract.ContractType.class,
+          long.class);
+      privateMethod.setAccessible(true);
+      privateMethod.invoke(wallet, message, contractType, timeout);
+
+      assertTrue(true);
```

Comment:
What's the point here?

Reply:
This means that the unit test code runs here and there is no problem

Reply:
It can be removed. It's also fine to leave it here.

Reply:
I suggest deletion.